### ISSUE TITLE
Script tag usage in custom tracking

### ISSUE THREAD

**Issue by dmetzler1988** [sulu/sulu#3259]
| Q | A
| --- | ---
| Bug? | no
| New Feature? | yes

#### Actual Behavior

If you add a custom tracking option and you need a own script element with defer as example, you can't do this because sulu set a own script around it.

#### Expected Behavior

We need a option to add the whole script block in the custom tracking.

#### Steps to Reproduce

Add a custom tracking in webspace properties with a script block around it.
The website throws an error after reload.

#### Possible Solutions

Add a new option named like "custom tracking with own tags" in which you can add your own tags like scripts, html, php, ...

**Comment by danrot** [comment]
@dmetzler1988 What tracking code are you trying to integrate? It might also be an option to create another provider for that exact use case.

@trickreich You've had the same issue, right? Did you already clarify your situation? Must your code be present on all pages, or only on a single page?

**Comment by trickreich** [comment]
@danrot I think we have discussed this. It's a legit use case that you need to add a piece of code with html or other tags.

**Comment by trickreich** [comment]
and yes, the code must be present on all pages.

**Comment by trickreich** [comment]
another example:
```
        <!-- liveChat - Userlike -->
        <script type="text/javascript"src="//userlike-cdn-widgets.s3-eu-west-1.amazonaws.com/<ID>.js">
        </script>
```

**Comment by alexander-schranz** [comment]
fixed in 1.6.7
